cmdstanpy/__init__.py

```diff
@@ -1,10 +1,14 @@
 import atexit
+import os
 import os.path
 import shutil
 import tempfile
 
+if 'CMDSTAN' in os.environ:
```

Comment:
To be a little pedantic, it's preferable to encapsulate this sort of logic in a function and call that function instead of referencing a module global everywhere.  For one, once the module is imported, one can't debug the logic without reloading.

Reply:
In such a function, one'd also have an
```python
if not os.path.exists(cmdstan_path):
    raise CmdStanNotFound(cmdstan_path)
if not os.path.exists(os.path.join(cmdstan_path, 'bin', 'stanc')):
    warn("CmdStan does not seem to be built")
```
etc.  It's a little pedantic, but this already presented a problem on Travis since the tests failed and CmdStanPy didn't notice that CmdStan wasn't built.